experimental/SetPartitions/src/LinearSetPartition.jl

```diff
@@ -0,0 +1,235 @@
+"""
+    LinearSetPartition
+
+LinearSetPartition represents a linear combination of set-partitions. See Chapter 5 in [Gro20](@cite).
```

Comment:
This is not entirely true, as it can also represent a linear combination of spatial partitions etc.
I find this name a bit misleading, as it implies that this only wraps SetParition and not AbstractPartition. What do you think about e.g. LinearPartition{...}?